reana_cluster/cli/__init__.py

```diff
@@ -73,7 +73,7 @@ def cli(ctx, loglevel, skip_validation, file):
     logging.basicConfig(
         format=DEBUG_LOG_FORMAT if loglevel == 'debug' else LOG_FORMAT,
         stream=sys.stderr,
-        level=logging.DEBUG if loglevel == 'debug' else logging.WARNING)
```

Comment:
I refactored this because I tested and level INFO was never set.

Reply:
I am changing it in `reana-client` as well, thanks 👍